contracts/margin/external/ERC20/ProductionERC20Long.sol

```diff
@@ -0,0 +1,136 @@
+/*
+
+    Copyright 2018 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.4.24;
+pragma experimental "v0.5.0";
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import { ERC20Long } from "./ERC20Long.sol";
+import { MarginCommon } from "../../impl/MarginCommon.sol";
+import { MarginHelper } from "../lib/MarginHelper.sol";
+
+
+/**
+ * @title ProductionERC20Long
+ * @author dYdX
+ *
+ * Early production version of an ERC20Long with a token cap and a trusted closer
+ */
+contract ProductionERC20Long is
```

Comment:
`ProductionERC20Long` is a bad name. It makes it sound like all the other ones are just for dev. I think just `CappedERC20Long` is fine